src/helpers/navbar.js

```diff
@@ -0,0 +1,31 @@
+import {Link} from "react-router-dom"
+import React from 'react';
+import "./nav.css"
+import {FaHome,FaTable,FaChartLine, FaBookOpen} from "react-icons/fa"
```

Comment:
the naming of these components seem a bit odd, can you expand Fa or maybe just remove it all together?